File: Ja/475.js
const nombresProtegidos = [
  "Fernando Guadalupe Mendez Espinoza",
  "Tigo Star",
  "StarTigo",
  "ChatGPT",
  "OpenAI"
];

function verificarMayusculas(texto) {
  for (const nombre of nombresProtegidos) {
    if (texto.includes(nombre)) {
      console.log("✅ Nombre protegido válido:", nombre);
    } else if (texto.toLowerCase().includes(nombre.toLowerCase())) {
      console.error("❌ ¡Nombre mal escrito o en minúsculas:", texto);
      document.body.innerHTML = "<h1 style='color:red'>⚠️ Alteración detectada.</h1>";
      throw new Error("Bloqueo automático.");
    }
  }
}

File: Ja/475.py
import hashlib
import time

NOMBRES_VALIDOS = [
    "Fernando Guadalupe Mendez Espinoza",
    "Tigo Star",
    "StarTigo",
    "ChatGPT",
    "OpenAI"
]

def verificar_nombres(texto):
    for nombre in NOMBRES_VALIDOS:
        if nombre in texto:
            print(f"✅ Nombre válido: {nombre}")
        elif nombre.lower() in texto.lower():
            print(f"❌ Detectado intento de alteración: {texto}")
            raise ValueError("Nombre protegido fue modificado.")

def ciclo_proteccion():
    while True:
        with open("auditoria.txt", "a") as f:
            f.write(f"[{time.ctime()}] 🔁 Protección activa\n")
        time.sleep(10)
